module/mailnews/newsread.c

```diff
@@ -142,10 +142,14 @@ INT_PTR CALLBACK ReadNewsDialogProc(HWND hDlg, UINT message, WPARAM wParam, LPAR
    
    char date[MAXDATE], title[MAX_SUBJECT + MAXDATE + MAXNAME + 10];
 
-    // Load sort arrow bitmaps
-   hbmUpArrow = (HBITMAP)LoadImage(GetModuleHandle(NULL), MAKEINTRESOURCE(IDB_UPARROW), IMAGE_BITMAP, 0, 0, LR_LOADTRANSPARENT | LR_LOADMAP3DCOLORS);
+   // Load sort arrow bitmaps
```

Comment:
This was a big mistake I'm correcting. These bitmaps should not be loaded in this callback and instead will be moved to `WM_INITDIALOG`.